examples/llama2/README.md

```diff
@@ -36,6 +36,14 @@ Supported languages are Python, TypeScript, JavaScript, Ruby, Julia, Rust, C++,
 
 Requirements file: [requirements-qlora.txt](requirements-qlora.txt)
 
+### Inference optimization using ONNX Runtime GenAI
+For using ONNX runtime GenAI to optimize, follow build and installation instructions [here](https://github.com/microsoft/onnxruntime-genai).
+
+Run the following command to execute the workflow:
+```bash
+python -m olive.workflows.run --config lamma2_genai.json
```

Comment:
Is that possible to add a short code snippet to introduce how to run generation task with model from genai converter?

Reply:
Do you mean invoking the GenAI builder directly outside of Olive or using the generated onnx model?
If later, the output model is an onnx model so usage should be no different.